src/main/webapp/js/adminActivityLog.es6

```diff
@@ -6,6 +6,23 @@ $(document).ready(() => {
     $('#filterReference').toggle();
     bindBackToTopButtons('.back-to-top-left, .back-to-top-right');
     highlightKeywordsInLogMessages();
+
+    $('#toggleReferenceLink').on('click', (e) => {
+        e.preventDefault();
```

Comment:
For instance, if it is an anchor tag and we want to prevent the default, that really doesn't make sense, we should just make it a button instead of an anchor tag, we have a lot of these prevent default and a corresponding ```return false;``` in our JavaScript code and that is not a good pattern